### ISSUE TITLE
npm install errors with "computed checksum did NOT match"

### ISSUE THREAD

**Issue by jakebailey** [dprint/dprint#367]
I'm trying to use `npm` to install `dprint` (as a replacement to prettier), but the install appears to fail.

```
> dprint@0.15.0 postinstall /home/jake/testproj/dprinttest/node_modules/dprint
> node ./install.js

######################################################################################################################################################## 100.0%
dprint.zip: FAILED
sha256sum: WARNING: 1 computed checksum did NOT match
node:child_process:747
    throw err;
    ^

Error: Command failed: /home/jake/testproj/dprinttest/node_modules/dprint/install.sh 0.15.0 18be4651601822e425de7f536e56f5095b257ffc0fdc019759bd967d8f888e1f
    at checkExecSyncError (node:child_process:707:11)
    at Object.execSync (node:child_process:744:15)
    at Object.<anonymous> (/home/jake/testproj/dprinttest/node_modules/dprint/install.js:34:23)
    at Module._compile (node:internal/modules/cjs/loader:1109:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:10)
    at Module.load (node:internal/modules/cjs/loader:989:32)
    at Function.Module._load (node:internal/modules/cjs/loader:829:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
    at node:internal/main/run_main_module:17:47 {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 120592,
  stdout: null,
  stderr: null
}
npm WARN dprinttest@1.0.0 No description
npm WARN dprinttest@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dprint@0.15.0 postinstall: `node ./install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the dprint@0.15.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/jake/.npm/_logs/2021-06-15T17_05_46_070Z-debug.log
```

**Comment by dsherret** [comment]
Oops... I looked through my command history and I accidentally published the 0.15.0 package with the 0.14.0 checksums. My bad!

Usually the publish is done automatically on the CI, but I accidentally broke the publish script for the 0.15.0 release. I should have tried it out after publishing.

Anyway, this should be resolved now in the 15.0.2 npm package. Thanks very much for reporting!

**Comment by jakebailey** [comment]
Thank you!

**Comment by jakebailey** [comment]
I can confirm it works, though it surprised me to see 0.15.2 published with the underlying tool being 0.15.0.

**Comment by dsherret** [comment]
Yeah, I was thinking about that, but I was lazy to do a full rust build for the sake of the npm package and this saves a bit of energy 🙂

I do new releases every few weeks anyway, so it will be resolved soon.
